derivation/src/main/scala/caliban/derivation/annotation/GQLExclude.scala

```diff
@@ -0,0 +1,8 @@
+package caliban.derivation.annotation
+
+import scala.annotation.StaticAnnotation
+
+/**
+ * Annotation used to indicate that a class member should not be included in the derived Schema
+ */
+class GQLExclude extends StaticAnnotation
```

Comment:
How about adding this to the other annotations? It seems pretty simple to support it in the Magnolia case.

Reply:
Sure thing.